src/components/Header.js

```diff
@@ -153,13 +167,25 @@ const Header = ({ setContext }) => {
       additionalClasses="docs-masthead"
       utilityContents={utilityItems}
       navLink={
-        <SprkTextInput
-          additionalClasses="docs-header-search sprk-u-Width-100"
-          leadingIcon="search"
-          hiddenLabel
-          name="InlineSearch"
-          placeholder="Search"
-        />
+        // eslint-disable-next-line max-len
+        <SprkInputContainer additionalClasses="docs-header-search sprk-u-Width-100">
```

Comment:
Does this need the `sprk-u-Width-100`? I thought we added that to the input containers now. If we can, try not to use the utils since we are moving away from that approach.

Reply:
This needs it. We added the width: 100 to the `SprkInput` but not the container. I think this is fine to use the util since we aren't using it in our package components. I can add the width: 100 to the `docs-header-search` if it doesn't affect anything else, though.

Reply:
Yeah, lets add it to that. I think we only have `docs-header-search` that we use on the doc site. So should be fine. We want to model good UI development for teams in this move away from using utils especially if they are checking out how we do things. Showing them in our doc that you can build a site without the utils will be helpful.